runtime/rasdump/javadump.cpp

```diff
@@ -1054,12 +1054,30 @@ JavaCoreDumpWriter::writeEnvironmentSection(void)
 
 	/* NB : I can't find any code for making this decision in the existing implementation */
 #ifdef J9VM_BUILD_J2SE
+	/* If JITServer enabled, print running mode as JITServer and client UID*/
+#ifdef JITSERVER_SUPPORT
+	if (_VirtualMachine->internalVMFunctions->isJITServerEnabled(_VirtualMachine))
+		_OutputStream.writeCharacters("a JITServer Server");
+	else if (0 != jitConfig->clientUID)
+		_OutputStream.writeCharacters("a JITServer Client");
+	else
+		_OutputStream.writeCharacters("a standalone");
+#else /* !JITSERVER_SUPPORT */
 	_OutputStream.writeCharacters("a standalone");
-#else
-	_OutputStream.writeCharacters("an embedded");
+#endif 
+#else /* !J9VM_BUILD_J2SE */
+    _OutputStream.writeCharacters("an embedded");
 #endif
 	_OutputStream.writeCharacters(" JVM\n");
 
+#ifdef JITSERVER_SUPPORT
+	if (0 != jitConfig->clientUID) {
+		_OutputStream.writeCharacters("1CICLIENTID    Client UID "); 
+		_OutputStream.writeInteger64(jitConfig->clientUID, "%llu");
+		_OutputStream.writeCharacters("\n");
+	}
+#endif
```

Comment:
This should have a comment indicating the matching `#if` (i.e. `/* JITSERVER_SUPPORT */`).